utils/souper2llvm.in

```diff
@@ -611,10 +618,13 @@ def printFuncHeader():
             ptr = "%ptr_" + str(MDRangePtr)
             masks = "  " + ptr + " = alloca " + w + "\n" +\
             "  store " + w + " " + name + ", " + w + "* " + ptr + "\n" +\
+            "  call void @fun_" + str(functionDeclCount) + "(" + w + "* " + ptr + ")\n" +\
             "  " + old + " = load " + w + ", " + w + "* " + ptr + ", !range !" + str(MDRangePtr) + "\n" +\
             "  store " + w + " " + old + ", " + w + "* @glob_" + regnum + "_nonZero\n" + masks
             metadataStr = metadataStr + "!" + str(MDRangePtr) + " = " + "!{ " + w + " 1" + ", " + w + " 0"+ " }\n"
             MDRangePtr += 1
+            functionDeclStr += "declare void @fun_" + str(functionDeclCount) + "(" + w + "* " + ptr + ")\n";
```

Comment:
I'd suggest to take off the semi-colon at the end. Though it won't make a difference here in this case. Just a nitpick.